app/lib/features/onboarding/widgets/encryption_backup_widget.dart

```diff
@@ -0,0 +1,147 @@
+import 'dart:io';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_easyloading/flutter_easyloading.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:acter/common/providers/app_install_check_provider.dart';
+import 'package:acter/features/share/action/shareTo.dart';
+import 'package:phosphor_flutter/phosphor_flutter.dart';
+import 'package:share_plus/share_plus.dart';
+
+class PasswordManagerBackupWidget extends ConsumerWidget {
+  final String encryptionKey;
+
+  const PasswordManagerBackupWidget({
+    super.key,
+    required this.encryptionKey,
+  });
+
+  Future<void> _buildShareContent() async {
+    await Clipboard.setData(ClipboardData(text: encryptionKey));
+    EasyLoading.showSuccess('Key copied to clipboard');
+  }
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final availableApps = <Widget>[];
+
+    // Add copy button
+    availableApps.add(
+      _buildActionButton(
+        icon: Icons.copy,
```

Comment:
this would be more legible if we kept using the actual list and if directly:

```suggestion
  @override
  Widget build(BuildContext context, WidgetRef ref) => Wrap(
   //..
  children: [
    // Add copy button
      _buildActionButton(
        icon: Icons.copy,
   //...
   if (ef.watch(isAppInstalledProvider(ExternalApps.onePassword)).valueOrNull == true) 
          _buildActionButton(
            icon: Icons.security,
   
```

Reply:
Addressed by this [commit](https://github.com/acterglobal/a3/pull/2800/commits/b23de2a9fa981fcf14acc95c6a9f354286da42a4)